### DISCUSSION TITLE
Flottenmanager

### DISCUSSION THREAD

**Discussion by BTSVDanielk** [rjcncpt/StarCitizen-Deutsch-INI#553] [Übersetzungsvorschläge]
Ich glaube in der 3.22 hieß eurer Fleetmanager noch Vehikel Manager und jetzt Flottenmanager.... kann es sein das ihr vergessen habt im Ladebildschirm siehe Bild "Vehikelbestand wird aufgerufen"  es in "Flottenbestand" wird aufgerufen zu ändern" oder sollte es so wie es jetzt ist sein. ?

Gruß Daniel :)

![ScreenShot-2024-06-01_23-10-48-A96](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/8f818ed6-95b3-401a-a4ee-178d53e6c815)

**Comment by rjcncpt** [comment]
Hey 😁 Flottenbestand klingt natürlich besser. Wird geändert. Danke 😊

**Reply by MaxM1211** [reply]
Ich fände `Flotte wird abgerufen` besser.

**Reply by BTSVDanielk** [reply]
ja bei mir ging es nur um das Wort Vehikelbestand.... das in Flottenbestand geändert werden sollte. Ich habe "wird aufgerufen" falsch übertragen ... "Flotte wird abgerufen" klingt gut.

**Reply by BTSVDanielk** [reply]
Wenn ich ein Schiff Claime, dann steht da auch noch einmal Vehikel.



![ScreenShot-2024-06-02_01-13-53-AFB](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/14daf370-0b00-42d7-9b5d-ba257d1aa0fe)

**Reply by MaxM1211** [reply]
Soweit ich weiß muss das dort bleiben, da CIG keine Unterscheidung zwischen Schiff und (Boden)Fahrzeug macht. Deswegen haben wir an vielen Stellen Vehikel stehen.

**Reply by rjcncpt** [reply]
Jo. CIG schreibt auch Vehicle
`Once submitted, the following vehicle will be delivered to your current location in the estimated timeframe.`

> Ich fände `Flotte wird abgerufen` besser.

Ich auch.

**Reply by rjcncpt** [reply]
Ich hab die Änderungen mal in den PR auf dem privaten Github rein übertragen.

**Reply by BTSVDanielk** [reply]
Ok. Interessiert Ihr Euch auch für Texte, die nicht in Ihre vorhergesehenen Felder passen?
Die 12 schiesst ein wenig hinaus aus dem Feld.

![ScreenShot-2024-06-01_13-38-59-740](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/f0847c07-f1e6-4ab2-99a4-baa73b799ca1)

**Reply by MaxM1211** [reply]
@BTSVDanielk Grundsätzlich ja
Das Problem hier im Bild, werden wir aber kaum lösen können, denn das ist ein Ortsname, der überall verwendet wird und deswegen auch nicht abgekürzt werden kann.
Wie siehst du das @rjcncpt ?

Falls dir (@BTSVDanielk) aber sonst noch zu lange Texte auffallen, kannst gerne eine Neue Diskussion öffnen (machts übersichtlicher :-D)

Danke dir, und ich wünsche einen guten Start in die Woche morgen :-)

**Reply by rjcncpt** [reply]
> @BTSVDanielk Grundsätzlich ja Das Problem hier im Bild, werden wir aber kaum lösen können, denn das ist ein Ortsname, der überall verwendet wird und deswegen auch nicht abgekürzt werden kann. Wie siehst du das @rjcncpt ?
> 
> Falls dir (@BTSVDanielk) aber sonst noch zu lange Texte auffallen, kannst gerne eine Neue Diskussion öffnen (machts übersichtlicher :-D)
> 
> Danke dir, und ich wünsche einen guten Start in die Woche morgen :-)

Das betrifft nur die vollständige Übersetzung. Nicht die normale. Ich könnte da einen anderen Namen für finden, weil das ja eh ziemlich frei in der Übersetzung ist.

In der engl. wäre es `Shubin Processing Facility SPAL-12`

**Reply by rjcncpt** [reply]
`Fertigungsanlage` wäre 3 Zeichen kürzer

**Reply by BTSVDanielk** [reply]
Ok, dann weiß ich Bescheid. Euch auch einen guten Start in die neue Woche. Nächstes mal gibt es eine neue Diskussion. ;)

**Reply by rjcncpt** [reply]
> Ok, dann weiß ich Bescheid. Euch auch einen guten Start in die neue Woche. Nächstes mal gibt es eine neue Diskussion. ;)

Danke ebenso :)

Wie findest du Fertigungsanlage als Alternative oder hast du eine bessere?

Viele Grüße

**Reply by BTSVDanielk** [reply]
Produktionsstätte Produktionsanlage ist mir sofort eingefallen. Aber Fertigungsanlage hört sich besser an. Ich arbeite zwar nicht in der Produktion, aber bei Stellenausschreibungen steht bei uns auch heute nur Fertigung. Daher ist Fertigungsanlage schon ein gutes Wort.

Ich bin gestern 2 Stunden nicht ins Spiel gekommen, deswegen musste ich Star Citizen neu installieren. Ich habe jetzt die deutsche Übersetzung aus den PTU in das Live kopiert. Ich habe vergessen die alte global.ini zu sichern. Jetzt sind genau die Namen wieder in Englisch *g
